portal-api/src/v1/controllers/deal-status/user-can-submit-deal.js

```diff
@@ -1,5 +1,13 @@
+const { CHECKER, MAKER } = require('../../roles/roles');
+
 const userCanSubmitDeal = (deal, user) => {
-  const isMakerCheckerUser = (user.roles.includes('maker') && user.roles.includes('checker'));
+  const userRoles = user?.roles;
```

Comment:
And sorry if this is already covered in 1975

Reply:
Do you mean removing the `isMakerChecker` logic here? I've left it, assuming it would be covered by DTFS2-6637. I'll comment on the ticket now to make sure it's covered

Reply:
Ah yes! Great spot